cc/utils/cose/cwt_test.cc

```diff
@@ -61,9 +62,21 @@ class CertificateTest : public testing::Test {
 
 TEST_F(CertificateTest, CwtDeserializeSuccess) {
   auto cwt = Cwt::Deserialize(public_key_certificate_);
-  ASSERT_TRUE(cwt.ok()) << cwt.status();
+  EXPECT_TRUE(cwt.ok()) << cwt.status();
   EXPECT_THAT(cwt->subject_public_key.GetPublicKey(), ElementsAreArray(kTestPublicKey));
 }
 
+TEST_F(CertificateTest, CwtSerializeDeserializeSuccess) {
+  std::vector<uint8_t> test_public_key = {1, 2, 3, 4};
+  auto serialized_cwt = Cwt::SerializeHpkePublicKey(test_public_key);
+  EXPECT_TRUE(serialized_cwt.ok()) << serialized_cwt.status();
+  auto serialized_cwt_string = std::string(serialized_cwt->begin(), serialized_cwt->end());
+
+  auto deserialized_cwt = Cwt::Deserialize(serialized_cwt_string);
+  EXPECT_TRUE(deserialized_cwt.ok()) << deserialized_cwt.status();
```

Comment:
EXPECT_OK(serialized_cwt)
EXPECT_OK(deserialized_cwt)

I think this will print the status on failure.

Reply:
Looks like `EXPECT_OK` is not available externally:
```
cc/utils/cose/cwt_test.cc:72:3: error: 'EXPECT_OK' was not declared in this scope; did you mean 'EXPECT_EQ'?
   72 |   EXPECT_OK(serialized_cwt) << serialized_cwt.status();
      |   ^~~~~~~~~
      |   EXPECT_EQ
```